driver/src/main/java/org/neo4j/driver/v1/Path.java

```diff
@@ -61,7 +61,7 @@
     Node end();
 
     /** @return the number of segments in this path, which will be the same as the number of relationships */
-    long length();
+    int length();
```

Comment:
Oh wait I misread.. this is about Path not FieldAccess...hmm I guess same reasoning applies here.